include/rellic/AST/StructGenerator.h

```diff
@@ -30,16 +30,16 @@ class StructGenerator {
   void VisitFields(
       clang::RecordDecl* decl, llvm::DICompositeType* s,
       std::unordered_map<clang::FieldDecl*, llvm::DIDerivedType*>& map,
-      bool isUnion);
+      bool isUnion, int sizeHint);
```

Comment:
Can you name a custom type for this, using a `typedef` or a `using` statement? Then, comment that Clang's maximum structure alignment is `536870912` and the GCC's is `268435456`. Also, add some stress test test cases, e.g.
```c++
struct alignas(1ull << 32) omg {
    int x[1ull << 32];
};
```

Reply:
The maximum `alignas` I was able to get compiling was `1 << 29`